cmd/minishift/cmd/root.go

```diff
@@ -108,6 +108,17 @@ var RootCmd = &cobra.Command{
 		// creating all directories for minishift run
 		createMinishiftDirs(state.InstanceDirs)
 
+		// Ensure the viper config file exists.
+		ensureConfigFileExists(constants.ConfigFile)
+
+		// Read the config file and get the details about existing image cache.
+		// This should be removed after 3-4 release of minishift. [PK]
```

Comment:
@praveenkumar we do not need `[pk]` as it can be figured out using `git blame`. I will fix it while merging it.

Reply:
this is just to make sure someone commented it in case retrieve it back.

Reply:
@praveenkumar we do not need names in comments.  But as I mentioned no need to fix it as I can change it and then merge.